src/test/java/io/anserini/StdOutStdErrRedirectableLuceneTestCase.java

```diff
@@ -26,6 +26,16 @@ public abstract class StdOutStdErrRedirectableLuceneTestCase extends LuceneTestC
   protected PrintStream saveOut;
   protected PrintStream saveErr;
 
+  protected static void suppressJvmLogging() {
```

Comment:
Nit: Ideally this should be part of LuceneTestCase since now many of the test classes that don't need redirect are extending this class to supress logs...
Hpw about a two level class:SuppresedLoggingLuceneTestCase extends LuceneTestCase and StdOut... extends Suppressed....